spec/ruby/optional/capi/string_spec.rb

```diff
@@ -1236,6 +1236,14 @@ def inspect
     it "returns the same string as String#-@" do
       @s.rb_enc_interned_str_cstr("hello", Encoding::UTF_8).should.equal?(-"hello")
     end
+
+    ruby_version_is "3.4" do
```

Comment:
Since https://bugs.ruby-lang.org/issues/18772 has been backported, this fix should be backported too.
```suggestion
    ruby_bug "#20322", ""..."3.4" do
```